tests/monit/test_monit_service_status.py

```diff
@@ -0,0 +1,26 @@
+"""
+Test the running status of Monit service
+"""
+import logging
+
+import pytest
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any')
+]
+
+
+def test_monit_service_status(duthost):
```

Comment:
Suggest shortening the name of this function and the file to `test_monit_status`